first-analysis-steps/physics-at-lhcb.md

```diff
@@ -0,0 +1,147 @@
+
+All objects created by the reconstruction contain contributions from detector
+effects, such as resolution. In addition, not all created objects may
+correspond to real particles; we could for example create ‘ghost’ tracks,
+trajectories created from combinations of random hits that together _look like_
+a real trajectory.
+
+Given this, we can never know anything with complete certainty. Instead we
+must infer properties _statistically_ based on ensembles of objects and events.
+
+Let’s say we want to count the number of [$$J/\psi$$ mesons][pdgjpsi] that
+decay to two muons, $$\mu^{+}\mu^{-}$$. This proceeds via three steps:
+
+1. Select tracks created by the reconstruction;
+2. Create pairs of oppositely-charged tracks;
+3. Fit each pair under the hypothesis that they originate from a common point
+   in space.
+
+We’ll go over each of these steps, starting with tracks produced by the
+reconstruction. Naturally, there are many of these in any given event,
+typically hundreds, so we begin be applying _selections_ on the tracks based on
+our physics understanding. For example, the $$J/\psi$$ is quite heavy (at
+around $$3.1\,\mathrm{GeV}$$), so we might expect its decay products to have a
+higher momentum on average than objects produced from soft processes in the
+collision. The reconstruction gives us some probability-like
+information for a given track to have been created by a true muon, so we might
+require that the tracks we use have some minimum ‘muon probability’.
+
+With a reduced set of a tracks, we can create all pairs of opposite-sign muons.
+We know that a real particle decay happens at a point in space, so we could
+require that the distance of closest approach between the two muons does not
+exceed some maximum value. We could further require that the invariant mass of
+the dimuon combination is close to the known mass of the $$J/\psi$$, invoking
+the conservation of momentum.
+
+With selected dimuon pairs, we can _fit_ a $$J/\psi \to \mu^{+}\mu^{-}$$ decay
+vertex. This is done by expressing the hypothesis that there is a common origin
+vertex of both tracks as an optimisation problem, and then varying the measured
+$$\mu^{+}$$ and $$\mu^{-}$$ four-momenta within their measured uncertainties to
+best fit that hypothesis. The result is a vertex object which has, for example,
+a fit $$\chi^{2}$$ associated to it. The quality of the fit can be used in a
+selection.
+
+Finally, with the fitted muon momenta, we can form the four vector of the
+$$J/\psi$$ as their sum, creating the $$J/\psi$$ _candidate_[^1]. Now we get
+back to our original goal or measuring the number of true $$J/\psi \to
+\mu^{+}\mu^{-}$$. By plotting the $$J/\psi$$ invariant mass values as a
+histogram, we might hope to see a signal component.
+
+<a href="img/dimuon_mass.png"><img alt="Dimuon invariant mass spectrum" src="img/dimuon_mass.png" style="width: 50%; margin: 0 25%;"/></a>
+
+We can then use model the components using probability density functions, and
```

Comment:
This sentence looks not consistent. "We can then use model" -> "We can then model"?